README.md

```diff
@@ -48,6 +48,29 @@ Finally, try to keep your branches focused on fixing/adding only one feature and
 
 Here is a simplified workflow on how to add a new feature:
 
+### Syncing with fork
+
```

Comment:
Can you also mention somewhere that `upstream` and `origin` are just names holding the actual URLs of these git repositories

Reply:
Thanks for pointing that out, I have mentioned that in latest commit.